### ISSUE TITLE
Unexpected behavior with strings in ternaries

### ISSUE THREAD

**Issue by bkjohnson** [edvardchen/eslint-plugin-i18next#79]
It seems like ternary expressions that have two strings as the return options don't trigger the rule, but if only one of the options is a string then the rule will notice it.

![Rule throwing error on `"Continue" : undefined` ternary but not for `"Yes" : "No"` ternary](https://user-images.githubusercontent.com/2008881/178028249-9818f7f2-9778-4523-ad81-bdd88607e6a4.png)

**Comment by edvardchen** [comment]
@bkjohnson may I know the plugin's version and options ?

**Comment by bkjohnson** [comment]
Of course. The version is `6.0.0-4`, and the configuration options are:

```
  "rules": { 
     "i18next/no-literal-string": [ 
       2, 
       { 
         "mode": "all", 
         "jsx-attributes": { "include": ["aria-label", "text"] }, 
         "object-properties": { "exclude": ["tag", "styleUrl", "componentName", "eventName", "action"]}, 
         "words": { "exclude": ["", " ", "\\d", "-\\d", "tabindex.*", "vads.*", "aria-.*", "true", "false", "checked"] }, 
         "callees": { "exclude": ["i18next.*", "Listen", "consoleDevError",  "querySelector.*", "InputEvent", "Watch", "createElement", "removeAttribute", "getAttribute", "getElement.*", "NumberFormat", "CustomEvent", "SubmitEvent", "addEventListener", "getSlottedNodes", "add", "remove", "contains", "split", "classnames"] }, 
         "should-validate-template": true 
       } 
     ] 
   },
```

**Comment by bkjohnson** [comment]
I've created a [demo project](https://github.com/bkjohnson/eslint-i18next-demo) for testing. `yarn lint` will run the linting in case your editor doesn't show the warnings, but you should see that [these lines in the typescript source](https://github.com/bkjohnson/eslint-i18next-demo/blob/ce821acc9aca611c366e0da3bb6fd1f3bdb56bd8/src/components/my-component/my-component.tsx#L30-L33) demonstrates the issue.

```jsx
  render() {
    // No linting warnings on this ternary
    const foo = this.first === "John" ? "bar" : "baz";
    // Linting warnings on this ternary
    const foo2 = this.first === "John" ? "bar" : undefined;
    return (
      <div>
        <div>Hello, World! I'm {this.getText()}</div>
        <div>{foo} {foo2}</div>
      </div>
    );
  }
```

**Comment by edvardchen** [comment]
Fixed it in `6.0.0-5`
